### PR TITLE
Create smart meter reads API

### PR DESCRIPTION
**Pull request by arvindchez**
This PR adds new API endpoints for handling smart meter reads.

### CODE REVIEW

File: packages/origin-energy-api/src/reads/reads.controller.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        @Param('meter') meterId: string,
+        @Query() filter: AggregateFilterDTO
+    ): Promise<AggregatedReadDTO[]> {
+        return super.getReadsAggregates(meterId, filter);
+    }
+
+    @Post('/:meter')
+    @ApiBody({ type: MeasurementDTO })
+    @ApiCreatedResponse({ description: 'Creates meter reads' })
+    public async storeReads(
```
Comment (kosecki123) [inline packages/origin-energy-api/src/reads/reads.controller.ts:67]: Auth and authorization are missing here. For now please add `@UseGuards(AuthGuard())` to handle authentication, later we would need to add the check if the authenticated user is an owner of the device.

File: packages/origin-energy-api-client/package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
+    "name": "@energyweb/origin-energy-api-client",
```
Comment (kosecki123) [inline packages/origin-energy-api-client/package.json:2]: Please add `private: true` for now, we would remove it when we get it fully integrated into the rest of the system.

File: packages/origin-energy-api/package.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
+{
+    "name": "@energyweb/origin-energy-api",
+    "version": "0.0.1",
```
Comment (kosecki123) [inline packages/origin-energy-api/package.json]: Please add `private: true` for now, we would remove it when we get it fully integrated into the rest of the system.

File: packages/origin-energy-api/src/reads/reads.controller.ts (hunk 1 above)
Comment (arvindchez) [reply packages/origin-energy-api/src/reads/reads.controller.ts:67]: Fixed

File: packages/origin-energy-api-client/package.json (hunk 2 above)
Comment (arvindchez) [reply packages/origin-energy-api-client/package.json:2]: fixed

File: packages/origin-energy-api/package.json (hunk 3 above)
Comment (arvindchez) [reply packages/origin-energy-api/package.json]: fixed

**Pull request merged**